config/facebook_oauth.js

```diff
@@ -2,13 +2,13 @@ dotenv = require('dotenv');
 dotenv.config();
 module.exports = {
     development: {
-        client_id: '896021850846067',
```

Comment:
well just don't forget that this exists in the history of git, I got pasten because it suddenly stopped working for me..

Reply:
yeah sorry for that. it's in the new .env file. we need to design how we are going to handle all our environment variables